### PR TITLE
Fix documentation (follow up of the normalization PR #312)

### PR DESCRIPTION
**Pull request by HotLemonJuice**
Main changes:
- Fixed formatting errors for lists and fenced code blocks 
- Polished the wording

### CODE REVIEW

**Comment by HotLemonJuice** [comment]
Hello, @JavierJia ! I have confirmed all the changes using Jekyll.

File: docs/documentation.md [hunk 1]
```diff
@@ -3,89 +3,87 @@ layout: page
...
+
+```
+docker cp noah/src/main/resources/population/adm/allStatePopulation.adm nc1:/home
+docker cp noah/src/main/resources/population/adm/allCountyPopulation.adm nc1:/home
+docker cp noah/src/main/resources/population/adm/allCityPopulation.adm nc1:/home
+```
+
+* Open `noah/src/main/resources/population/sqlpp/ingestPopulation.sqlpp` using a text editor and copy all of its contents. Based on the IP address of nc1 and the path you copied population data into in the last step, you may need to modify the query a little bit, but in most of the cases you don't need to. Detailed instructions can be found in the comments of the queries. You can check the IP address of nc1 by running:
+
+{% raw %}
```
Comment (JavierJia) [inline docs/documentation.md]: it seems a new trick 😄 , can you share with us what `raw` is used for?

File: docs/documentation.md [hunk 2]
```diff
@@ -3,89 +3,87 @@ layout: page
...
-      ```bash
-      docker cp noah/src/main/resources/population/adm/allCityPopulation.adm nc1:/home
-      ```
-    - Open `noah/src/main/resources/population/sqlpp/ingestPopulation.sqlpp` using a text editor and copy all of its contents. Based on your nc1 IP address and the target path you copied population data into in the last step, you may need to change a little bit of the query, but in most of the cases you don't need to. Detailed instructions can be found in the comments of the queries. You can check the IP address of nc1 by running:
-      ```bash
-      docker inspect -f '{{range .NetworkSettings.Networks}}{{.IPAddress}}{{end}}' nc1
-      ```
-    - Go to `http://localhost:19001/ `. Copy and paste all the queries from the last step and click `run` with the default settings `SQL++` to ingest data into AsterixDB. After that, you should see `Success: Query Complete` in the output.
+* Ingest 33,107 US population data into AsterixDB (assume you are using Docker):
+* Run the following commands in the **cloudberry** folder to copy three population datasets into nc1
```
Comment (JavierJia) [inline docs/documentation.md]: into nc1 ==> into docker container `nc1`

File: docs/documentation.md [hunk 3]
```diff
@@ -3,89 +3,87 @@ layout: page
...
+
+```
+docker cp noah/src/main/resources/population/adm/allStatePopulation.adm nc1:/home
+docker cp noah/src/main/resources/population/adm/allCountyPopulation.adm nc1:/home
+docker cp noah/src/main/resources/population/adm/allCityPopulation.adm nc1:/home
+```
+
+* Open `noah/src/main/resources/population/sqlpp/ingestPopulation.sqlpp` using a text editor and copy all of its contents. Based on the IP address of nc1 and the path you copied population data into in the last step, you may need to modify the query a little bit, but in most of the cases you don't need to. Detailed instructions can be found in the comments of the queries. You can check the IP address of nc1 by running:
```
Comment (JavierJia) [inline docs/documentation.md]: Too many words for little information.
`Based on the IP address of nc1 and the path you copied population data into in the last step, you may need to modify the query a little bit, but in most of the cases you don't need to. Detailed instructions can be found in the comments of the queries.`
==> `You may need to change the IP address of  nc1 in the script.`

File: docs/documentation.md [hunk 4]
```diff
@@ -3,89 +3,87 @@ layout: page
...
+
+{% raw %}
+```
+docker inspect -f '{{range .NetworkSettings.Networks}}{{.IPAddress}}{{end}}' nc1
+```
+{% endraw %}
 
-6. Now that you have finished setting up the backend. Run cloudberry
+* Go to `http://localhost:19001/ `. Copy and paste all the queries from the last step and click **run** with the default settings **SQL++** to ingest data into AsterixDB. After that, you should see **Success: Query Complete** in the output.
```
Comment (JavierJia) [inline docs/documentation.md]:
```
Go to `http://localhost:19001/ `. Copy and paste all the queries from the last step and click **run** with the default settings **SQL++** to ingest data into AsterixDB. After that, you should see **Success: Query Complete** in the output.
```
==> 
```
Go to AsterixDB web interface (`http://localhost:19001/` in docker case). Copy and paste the `ingestPopulation.sqlpp` content into the web page and execute the query. You should see **Success: Query Complete** in the output.
```

File: docs/documentation.md [hunk 5]
```diff
@@ -3,89 +3,87 @@ layout: page
...
+* Now that you have finished setting up the backend. Run cloudberry
```
Comment (JavierJia) [inline docs/documentation.md]: ~~Now that you have finished setting up the backend.~~

File: docs/documentation.md [hunk 6]
```diff
@@ -3,89 +3,87 @@ layout: page
...
 
-7. Once cloudberry has successfully launched, register `ds_tweet` and `dsState/County/CityPopulation` data models into Cloudberry (assuming your working directory is still `cloudberry`).
+* Once Cloudberry has successfully launched, register data models into Cloudberry by running the following scripts in the **cloudberry** folder
 
 ```
 ./script/registerTwitterMapDataModel.sh
 ```
 
-7. (Optional) You can also deregister these data models from the Cloudberry to experiment with other data models. But be careful when doing this as the TwitterMap won't work without these data models mentioned above.
+* (Skip this step if you are setting up for the first time) You may also deregister these data models from the Cloudberry to experiment with other data models. But be careful when doing this as the TwitterMap won't work without these data models.
```
Comment (JavierJia) [inline docs/documentation.md]: move this part to the `Advanced` section.

File: docs/documentation.md (hunk 1 above)
Comment (HotLemonJuice) [inline docs/documentation.md]: It's for escaping double curly brackets. https://stackoverflow.com/questions/24102498/escaping-double-curly-braces-inside-a-markdown-code-block-in-jekyll

**Pull request merged**